api/scheme.go

```diff
@@ -47,6 +52,27 @@ var (
 )
 
 func init() {
+	capiaws.AddToScheme(InstallScheme)
```

Comment:
I'm not really proud of this implementation, but not sure if there's a better way

Reply:
wouldn't this be a matter of leaving this as it is, and then let the install package to unconditionally override with prometheusoperatorv1.AddToScheme(Scheme) after importing api/?

Also tangentially to this PR we have this duplicated in support/api/scheme. We need to consolidate and have it only once.

Reply:
I'm not sure that calling AddToScheme again will override the first one. For now, I think this is sufficient to get the behavior we want with the cli.